src/main/java/istjbot/task/Deadline.java

```diff
@@ -0,0 +1,51 @@
+package istjbot.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
```

Comment:
You might want to arrange the import statements in alphabetical order. The same issue is found is some of the other files as well. :)
```suggestion
import java.time.format.DateTimeFormatter;
import java.time.format.DateTimeParseException;
import java.time.LocalDate;
```